test/unit/mixins/evented.test.js

```diff
@@ -0,0 +1,455 @@
+/* eslint-env qunit */
+import sinon from 'sinon';
+import evented from '../../../src/js/mixins/evented';
+import * as Dom from '../../../src/js/utils/dom';
+import * as Obj from '../../../src/js/utils/obj';
+
+QUnit.module('mixins: evented', {
+
+  beforeEach() {
+    this.targets = {};
+  },
+
+  afterEach() {
+    Object.keys(this.targets).forEach(k => this.targets[k].trigger('dispose'));
+  }
+});
+
+QUnit.test('evented() mutations', function(assert) {
+  const target = this.targets.a = {};
+
+  assert.strictEqual(typeof evented, 'function', 'the mixin is a function');
+  assert.strictEqual(evented(target), target, 'returns the target object');
+
+  assert.ok(Obj.isObject(target), 'the target is still an object');
+  assert.ok(Dom.isEl(target.eventBusEl_), 'the target has an event bus element');
+  assert.strictEqual(typeof target.off, 'function', 'the target has an off method');
+  assert.strictEqual(typeof target.on, 'function', 'the target has an on method');
+  assert.strictEqual(typeof target.one, 'function', 'the target has a one method');
+  assert.strictEqual(typeof target.trigger, 'function', 'the target has a trigger method');
+});
+
+QUnit.test('evented() with exclusions', function(assert) {
```

Comment:
I don't think we ought to loop over all possible exclusions because that's... a lot. Especially when you consider that it can be an array of zero to four elements. I'll change it to be a reasonable subset.